lib/galaxy/visualization/plugins/interactive_environments.py

```diff
@@ -199,7 +199,7 @@ def get_conf_dict(self):
         }
 
         web_port = self.attr.galaxy_config.galaxy_infrastructure_web_port
-        conf_file['galaxy_web_port'] = web_port or self.attr.galaxy_config.guess_galaxy_port()
+        conf_file['galaxy_web_port'] = web_port
```

Comment:
It seems `self.attr.galaxy_config.galaxy_infrastructure_web_port` can be `None` (see https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/__init__.py#L583-L584 ), but also the (now removed) `guess_galaxy_port()` could return `None`, so I guess it is not a problem here?

Reply:
`galaxy_config.galaxy_infrastructure_web_port`  won't be `None` unless explicitly set to `None` after this is approved: https://github.com/galaxyproject/galaxy/pull/8795/commits/e50807a7c6ce050cedab623546f43d1b2b75ac52 

@jmchilton (or @natefoo) can you please take a look at that commit (url + web_port edit): I've described it in the commit message, and I'm not absolutely sure it won't cause any issues (it shouldn't, I think)

Reply:
@nsoranzo I've changed the milestone to 19.09, so yes, I think regardless of the edits I referred to in the other PR, this shouldn't be a problem. It's `None` or the explicitly set `web_port` (or, later, the default `web_port` after that other PR).